responsive-web-design-piano/css/style22.css

```diff
@@ -0,0 +1,60 @@
+/* box model reset */
+html {
+    box-sizing: border-box;
+}
+
+
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+
+
+/* piano styles */
+#piano {
+    width: 992px;
+    height: 290px;
+    margin: 80px auto;
+    padding: 90px 20px 0 20px;
+    background-color: #00471b;
+}
+
+
+.keys {
+    width: 949px;
+    height: 180px;
+    padding-left: 2px;
+    background-color: #040404;
+}
+
+
+.key {
+    position: relative;
+    width: 41px;
+    height: 175px;
+    margin: 2px;
+    float: left;
+    background-color: #ffffff;
+}
+
+
+.logo {
+    width: 200px;
+    position: absolute;
+    top: 23px;
+}
+
+
+/* pseudo elements */
+.key.black--key::after {
+    content: "";
+    position: absolute;
+    left: -18px;
+    width: 32px;
+    height: 100px;
+    background-color: #1d1e22;
+}
+
+/* 
+    The `<img>` element needs it's parent to have a position set for a
```

Comment:
The `<img>` element needs it's parent to have a position set as
point of reference. In the `#piano` rule, set the `position` to `relative`.